npeps/npep-28-egress-traffic-control.md

```diff
@@ -0,0 +1,74 @@
+# NPEP-28: Add northbound traffic support in (B)ANP API
```

Comment:
We just need to align this to the new NPEP format

Reply:
Well I saw you made an issue THX

Reply:
yes going to use https://github.com/kubernetes-sigs/network-policy-api/issues/126
let me update this.